### ISSUE TITLE
Current version detection from Podfile.lock when not present fixed version on Podfile.

### ISSUE THREAD

**Issue by r-plus** [renovatebot/renovate#7621]
**What would you like Renovate to be able to do?**

I started using renovate to update cocoapods managed dependencies but renovate will not provide any pull request.
In the renovate dashboard, it show the reason why not open the PR.
`"skipReason": "unknown-version"`

Yes, i only wrote dependency name in the Podfile like this.

```
pod 'SwiftLint'
```

I only locking the version by Podfile.lock file.

**Did you already have any implementation ideas?**

If no current version detectable by Podfile, parse the Podfile.lock to detect current version.

**Are there any workarounds or alternative ideas you've tried to avoid needing this feature?**

write version to Podfile.

**Is this a feature you'd be interested in implementing yourself?**

**Comment by rarkins** [comment]
This probably also requires #7623, which is applicable to any manager using update-lockfile strategy

**Comment by r-plus** [comment]
thanks for creating issue for manager wide architecture :)

**Comment by quoc-huynh-cosee** [comment]
Any updates on this?

**Comment by rarkins** [comment]
Hi, this still needs a minimal reproduction repo.

Also I'm not sure that `>= 0` is a good idea - we don't need that for others

**Comment by maoyama** [comment]
The minimal reproduction is here: https://github.com/maoyama/SampleSkipUnspecifiedVersion.

**Comment by rarkins** [comment]
@maoyama thanks. Are you able to manually create an example Pull Request updating the dependency, and describe which commands were used to do so?

**Comment by maoyama** [comment]
@rarkins  I created a Pull Request: https://github.com/maoyama/SampleSkipUnspecifiedVersion/pull/3. 
The only command I used was `pod update` .

**Comment by rarkins** [comment]
If you used `pod update`, wouldn't that update _every_ dependency in the lock file? This is not what Renovate should do

**Comment by maoyama** [comment]
@rarkins Yes, that's correct. I used `pod update Fakery` and created a Pull Request that updates only Fakery. https://github.com/maoyama/SampleSkipUnspecifiedVersion/pull/4.

**Comment by rarkins** [comment]
Does `pod update` update the Podfile, or only the lock file? Does it support updating to a _specific_ version, or only to the latest matching version?

**Comment by maoyama** [comment]
> Does pod update update the Podfile, or only the lock file?

only the lock file.

> Does it support updating to a specific version, or only to the latest matching version?

only to the latest matching version. 

ref:https://guides.cocoapods.org/using/pod-install-vs-update.html
